src/components/chart/_chart.scss

```diff
@@ -21,6 +22,55 @@
         line-height: 1.2rem;
         color: var(--ons-color-grey-75);
     }
+
+    &__footnote-number {
+        display: flex;
+        flex-shrink: 0;
+        flex-grow: 0;
+        align-items: center;
+        justify-content: center;
+        min-width: 1.125rem; // 18px
+        min-height: 1.125rem; // 18px
+        padding: 3px;
+        aspect-ratio: 1;
+        border: 1px solid var(--ons-color-grey-100);
+        border-radius: 50%;
+        font-size: 0.875rem; // 14px
+        line-height: 0;
+        background-color: var(--ons-color-white);
+        color: var(--ons-color-grey-100);
+    }
+
+    /* Override Highcharts styling */
+    &__footnote-number span {
+        position: static !important;
+        transform-origin: 0 0 !important;
+        font-size: inherit !important;
+        color: inherit !important;
+        top: 0 !important;
+        left: 0 !important;
+    }
+
+    &__footnotes {
+        list-style: none;
+        padding: 0;
+        margin: 0 0 1rem;
+
+        // We need to use a container query here in order to match the responsive rules for Highcharts
+        // which is based on the width that the chart renders not the viewport
+        // Update this is the Highcharts responsive rules change
+        @container (min-width: 601px) {
```

Comment:
```suggestion
        @media (min-width: 601px) {
```

Reply:
Hi @precious-onyenaucheya-ons we definitely need a [container query](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries) here. The reason is because Highcharts responsive rules are not responsive to the viewport, but to the width of the chart. Therefore in hiding and showing the annotations footnotes, we also need to respond to the width of the chart - otherwise there will be a screen size where either we get the numbers in the chart, but no footnotes below, or we get the text in the chart as well as below.

Reply:
I'm happy to resolve this since the minimum width is applied to the chart itself rather than the viewport.